logcat-verbose.sh

```diff
@@ -0,0 +1,7 @@
+adb shell setprop log.tag.BackupManagerService VERBOSE
```

Comment:
Could add a shebang to the first line so this can be run directly

`#!/bin/sh`

Reply:
Can this not be run directly without this as well? At least it works here (without forcing a choice of shell on you).

Reply:
Yeah but usually with scripts it's best to have a shebang set.